db/migrate/20160309141002_create_team.rb

```diff
@@ -0,0 +1,9 @@
+class CreateTeam < ActiveRecord::Migration
+  def change
+    create_table :team do |t|
+    	t.string :team
```

Comment:
It might be better if you name this `name` instead of `team` actually. When you reference the team, it's going to end up being code like this `Team.find(2).team`. Instead it might be better if we can write this: `Team.find(2).name` :)